lib/services/nativescript-migration-service.ts

```diff
@@ -97,12 +105,21 @@ export class NativeScriptMigrationService implements IFrameworkMigrationService
 				let projectType = this.$project.isTypeScriptProject().wait() ? NativeScriptMigrationService.TYPESCRIPT_ABBREVIATION : NativeScriptMigrationService.JAVASCRIPT_ABBREVIATION;
 				let pathToNewTnsModules = path.join(this.tnsModulesDirectoryPath, projectType, this.getFileNameByVersion(newVersion));
 				this.$fs.unzip(pathToNewTnsModules, tnsModulesProjectPath).wait();
+				if(!this.$fs.exists(appResourcesRequiredPath).wait() && this.$fs.exists(appResourcesObsoletePath).wait()) {
+					this.$logger.info(`Moving ${appResourcesObsoletePath} to ${appResourcesRequiredPath}`);
```

Comment:
I'm not sure this should be info message. What do you think, @teobugslayer , @Fatme ?